### PR TITLE
75 declare winner

### PR DESCRIPTION
**Pull request by oneorangecat**
Completed features detailed in #75, plus a minor typo fix in playbox_admin.module (was failing silently)

### CODE REVIEW

File: sites/all/modules/playbox_battles/playbox_battles.module [hunk 1]
```diff
@@ -118,11 +118,13 @@ function playbox_battles_theme_registry_alter(&$theme_registry) {
 }
 
 /**
- * Updates the vote total for a battler
+ * Updates the vote total for a battler and closes vote when winner
+ * has been declared
```
Comment (RobLoach) [inline sites/all/modules/playbox_battles/playbox_battles.module]: Function API documentation should be just a single line. If it exceeds that line, then you can have an additional paragraph underneath. https://www.drupal.org/coding-standards/docs#drupal

Perhaps just....

> Updates the vote total for a battler.
>
> When the winner has been declared, will close the vote.

**Comment by RobLoach** [comment]
Looks really good! There are some merge conflicts, however. GitHub provides the documentation to resolve them below. I'll be available next week if you would like assistance going through it.

Thanks a lot! Coming along nicely.

File: sites/all/modules/playbox_admin/playbox_admin.module [hunk 2]
```diff
@@ -84,7 +84,7 @@ function playbox_admin_views_pre_view(&$view) {
       //use default favorite president
       $playbox_fav_president_var = variable_get('playbox_admin_president');
       if(!empty($playbox_fav_president_var)){
-        $playbox_fav_president = $playbox_fav_president['entity_id'];
+        $playbox_fav_president = $playbox_fav_president_var['entity_id'];
```
Comment (RobLoach) [inline sites/all/modules/playbox_admin/playbox_admin.module]: This is a good bug fix. Good find.

**Comment by pirog** [comment]
Nice! Thanks again @RobLoach! Always great to have your discerning eye.

**Comment by pirog** [comment]
Merge conflict was on me so i've resolved that and also tested the func here. ENGAGE MERGE MINOTAUR!

File: sites/all/modules/playbox_battles/playbox_battles.module (hunk 1 above)
Comment (oneorangecat) [inline sites/all/modules/playbox_battles/playbox_battles.module]: Thanks Rob - was trying to tidy up comments but that one slipped through!

**Pull request merged**